packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -52,33 +32,21 @@ function WrappedNotificationsButton(
 
   useOutsideAlerter(wrapperRef, bellRef, close);
 
-  const { setWallet, setNetwork, setRpcUrl, saveLastReadMessage } = useApi();
-  const isWalletConnected = connected(props.wallet);
+  // TODO rewrite with new provider
```

Comment:
Is this temporary?

Reply:
yes, probably will refactor while refactoring react-ui package